### DISCUSSION TITLE
The Finals - Ai bot setup and the config model for The Finals

### DISCUSSION THREAD

**Discussion by Deeivi** [Babyhamsta/Aimmy#1308] [General]
Does anybody have the Ai bot setup and the config model for The Finals? 

Would appreciate very much

**Comment by Alex-test-mod** [comment]
setup  model

![labels_correlogram](https://github.com/Babyhamsta/Aimmy/assets/34565256/0c605cbc-7a34-48a8-b0d4-b514635da2c7)
![labels](https://github.com/Babyhamsta/Aimmy/assets/34565256/f706a55f-b2e2-4f00-a1db-0444904e616b)
![results](https://github.com/Babyhamsta/Aimmy/assets/34565256/3510a964-f326-433a-b59f-6c66c07b0e78)

**Reply by Alex-test-mod** [reply]
apex (:
